runtime/integration-tests/src/xcm/development/tests/connectors.rs

```diff
@@ -1650,8 +1647,19 @@ mod utils {
 	pub const DEFAULT_OTHER_DOMAIN_ADDRESS: DomainAddress =
 		DomainAddress::EVM(MOONBEAM_EVM_CHAIN_ID, [0; 20]);
 
+	pub const DEFAULT_MOONBEAM_LOCATION: MultiLocation = MultiLocation {
+		parents: 1,
+		interior: X1(Parachain(PARA_ID_MOONBEAM)),
+	};
 	pub type ConnectorMessage = Message<Domain, PoolId, TrancheId, Balance, Rate>;
 
+	pub fn get_default_moonbeam_native_token_location() -> MultiLocation {
+		MultiLocation {
+			parents: 1,
+			interior: X2(Parachain(PARA_ID_MOONBEAM), general_key(&[0, 1])),
+		}
+	}
+
```

Comment:
This can also be confusing since that this is not actually GLMR's multilocation. Also, there's no "default" location of a token, as that sounds like it may have multiple but this is the default one.

Reply:
It's the same location that we use to have in the `setup_pre_requirements`. Given that all the other const are prepended with `DEFAULT` I thought it makes sense to do the same here, even though it would be clearer if they were named `DEFAULT_TEST_*`